### DISCUSSION TITLE
Adjusting Volmue with a switch ?

### DISCUSSION THREAD

**Discussion by Billy60** [FrSkyRC/ETHOS-Feedback-Community#4707] [General]
Hi all, does anyone know how to adjust the Volume of vocal messages with a 3 pos switch on a Tandem XE ? It was easy and straightforward with a Horus X10e but in this new one I can't find the way ...
Also, Bertrand, I could find the 'invert' option for a condition with the X10e but on the XE cannot find it ? is it only for 'functions' ? I am trying to use it for mixes triggered via a 3 pos switch. 
Thanks for your input !

**Comment by bsongis** [comment]
You need to setup the volume levels for the 3 positions of your switch. I would use a VAR for that

**Reply by Billy60** [reply]
OK, how do you link the 'Vol' parameter to the VAR or switch position ? It was very easy on H10ex ...

**Reply by Billy60** [reply]
Sorry Bertrand ! found it ! a long press on enter ! clear !
Thanks again !!

**Comment by bsongis** [comment]
Invert is available when you long press [Enter] when you see an Hamburger icon on your Source

**Reply by Billy60** [reply]
This seems to invert the source, not the condition. I want to have a condition such as: "Ail > Dir for all S1 switch position except S1+"
Do I get it correctly ?

**Reply by bsongis** [reply]
Do you use a Logic Switch? You may invert it directly in the Logic Switch definition, or when you use it, again with the Hamburger icon.
